src/import/ImportAUP.cpp

```diff
@@ -1196,22 +1145,17 @@ bool AUPImportFileHandle::HandleSimpleBlockFile(XMLTagHandler *&handler)
 {
    FilePath filename;
    size_t len = 0;
-   
-   while (*mAttrs)
-   {
-      const wxChar *attr =  *mAttrs++;
-      const wxChar *value = *mAttrs++;
-
-      if (!value)
-      {
-         break;
-      }
 
-      const wxString strValue = value;
+   for (auto pair : mAttrs)
+   {
+      auto attr = pair.first;
+      auto value = pair.second;
 
       // Can't use XMLValueChecker::IsGoodFileName here, but do part of its test.
-      if (!wxStricmp(attr, wxT("filename")))
```

Comment:
I notice a few case insensitive comparisons in the old code.  Maybe it doesn't matter.

Some of the other ones are only about very old file formats.

Reply:
Thank you pointing out, fixed!